_delb/nodes.py

```diff
@@ -645,7 +645,7 @@ def __getitem__(self, item: Union[str, slice]) -> Optional[Attribute]:
             return None
 
     def __iter__(self) -> Iterator[str]:
-        yield from self._element.attrib  # type: ignore
+        return iter(self._element.attrib)  # type: ignore
```

Comment:
why tho? i don't really see why it would need fixing..?

Reply:
I mean i can iterate over a `TagNode`'s `.attributes` as is, so how is this a bug?

Reply:
the [https://docs.python.org/3/reference/datamodel.html#object.__iter__](method) shall return an [iterator](https://docs.python.org/3/glossary.html#term-iterator) object, and these need to implement an `__iter__` method as well which afaict isn't the case yet.

Reply:
alright